### ISSUE TITLE
worker depends on -bare Perl but it needs packages in the full fat Perl.

### ISSUE THREAD

**Issue by ehiggs** [hpcugent/easybuild-easyconfigs#2845]
[This line](https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/w/worker/worker-1.5.1-intel-2015a.eb#L18) should not have the `-bare`

The current install yields:

```
$ wsub -h
Can't locate Config/General.pm in @INC (you may need to install the Config::General module) (@INC contains: /apps/gent/SL6/sandybridge/software/Perl/5.22.1-intel-2016a-bare/lib/perl5/site_perl/5.22.1/x86_64-linux-thread-multi /apps/gent/SL6/sandybridge/software/Perl/5.22.1-intel-2016a-bare/lib/perl5/site_perl/5.22.1 /apps/gent/SL6/sandybridge/software/Perl/5.22.1-intel-2016a-bare/lib/perl5/5.22.1/x86_64-linux-thread-multi /apps/gent/SL6/sandybridge/software/Perl/5.22.1-intel-2016a-bare/lib/perl5/5.22.1 .) at /apps/gent/SL6/sandybridge/software/worker/1.5.1-intel-2015a/bin/wsub line 6.
BEGIN failed--compilation aborted at /apps/gent/SL6/sandybridge/software/worker/1.5.1-intel-2015a/bin/wsub line 6.
```

Swapping for the full fat version fixes it:

```
$ module swap Perl/5.22.1-intel-2016a
$ wsub -h
### error: batch file template should be specified
### usage: wsub  -batch <batch-file>          \
#                [-data <data-files>]         \
#                [-prolog <prolog-file>]      \
#                [-epilog <epilog-file>]      \
#                [-log <log-file>]            \
#                [-mpiverbose]                \
#                [-master]                    \
#                [-threaded]                  \
#                [-dryrun] [-verbose]         \
#                [-quiet] [-help]             \
#                [-t <array-req>]             \
#                [<pbs-qsub-options>]
#
#   -batch <batch-file>   : batch file template, containing variables to be
#                           replaced with data from the data file(s) or the
#                           PBS array request option
#   -data <data-files>    : comma-separated list of data files (default CSV
#                           files) used to provide the data for the work
#                           items
#   -prolog <prolog-file> : prolog script to be executed before any of the
#                           work items are executed
#   -epilog <epilog-file> : epilog script to be executed after all the work
#                           items are executed
#   -mpiverbose           : pass verbose flag to the underlying MPI program
#   -verbose              : feedback information is written to standard error
#   -dryrun               : run without actually submitting the job, useful
#   -quiet                : don't show information
#   -help                 : print this help message
#   -master               : start an extra master process, i.e.,
#                           the number of slaves will be nodes*ppn
#   -threaded             : indicates that work items are multithreaded,
#                           ensures that CPU sets will have all cores,
#                           regardless of ppn, hence each work item will
#                           have <total node cores>/ppn cores for its
#                           threads
#   -t <array-req>        : qsub's PBS array request options, e.g., 1-10
#   <pbs-qsub-options>    : options passed on to the queue submission
#                           command
```

**Comment by boegel** [comment]
@ehiggs: how about a PR to fix it?

**Comment by boegel** [comment]
fix in #2884

**Comment by ehiggs** [comment]
:+1:
